var/www/skitter.fanfic.dev/index.html

```diff
@@ -1,88 +1,111 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang=en>
 	<head>
 		<meta charset="utf-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Do you happen to have a reference for this?

Reply:
It turns out that piece of code isn't even necessary anymore. 
 https://stackoverflow.com/questions/6771258/what-does-meta-http-equiv-x-ua-compatible-content-ie-edge-do
 > November 2021 Update
>
> As this answer is now 10+ years old my recommendation would be to leave this tag out altogether unless you must support old legacy browsers.

Reply:
Let's drop it then :)  It already works in `links`, not sure there's much point supporting ancient IE if it can't even come out on top of that.

Reply:
Done :heavy_check_mark: